src/go/guestagent/pkg/tracker/apitracker.go

```diff
@@ -92,9 +88,21 @@ func (a *APITracker) Add(containerID string, portMap nat.PortMap) error {
 				continue
 			}
 
-			log.Debugf("calling %s API for the following port binding: %+v", exposeAPI, portBinding)
+			if a.enableListeners {
+				hostPort, err := strconv.Atoi(portBinding.HostPort)
+				if err != nil {
+					log.Errorf("error converting hostPort: %s", err)
+					continue
```

Comment:
This also stops trying to call gVisor; is that the desired behaviour? (Same with the other error in the `enableListeners` block, and in `Remove`.)